Lib/test/test_decimal.py

```diff
@@ -5817,6 +5842,7 @@ def doit(ty):
   CFunctionality,            PyFunctionality,
   CWhitebox,                 PyWhitebox,
   CIBMTestCases,             PyIBMTestCases,
+  CDecimalTypesTest,
```

Comment:
Wow, **decimal** must be the only remaining stdlib module with such an ancient test setup. I remember Serhiy cleaning up and modernising the stdlib test suite a couple of years ago; I guess something blocked that change from happening for the decimal module.

Reply:
cc. @serhiy-storchaka, do you remember the details?

Reply:
IIRC, this is the issue: gh-89392

Reply:
Ah, see https://github.com/python/cpython/issues/89392#issuecomment-1128465649